pkg/clusterresource/builder.go

```diff
@@ -104,9 +104,6 @@ type Builder struct {
 
 	// MachineNetwork is the subnet to use for the cluster's machine network.
 	MachineNetwork string
-
-	// SkipMachinePoolGeneration is set to skip generating MachinePool objects
-	SkipMachinePoolGeneration bool
```

Comment:
Why are we removing the option to skip machine pool generation?

Reply:
This was added in the interim while we waited for openstack machine pool support. I don't know of an actual use case to keep this, and if there's no use case it's a path I'd rather remove. A developer or advanced hiveutil user can do this by editing the -o yaml output.